bootstrap/kubeadm/internal/webhooks/kubeadmconfig_test.go

```diff
@@ -335,58 +340,44 @@ func TestKubeadmConfigValidate(t *testing.T) {
 			},
 			expectErr: true,
 		},
```

Comment:
Same here, why did we drop this test?

Reply:
This got dropped as the `UseExperimentalRetryJoin` is deprecated.